<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsusWidget</class>
 <widget class="QWidget" name="AsusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_fwdnx">
           <property name="text">
            <string>fwdnx </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fwimage">
           <property name="text">
            <string>fwimage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_osdnx">
           <property name="text">
            <string>osdnx </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_osimage">
           <property name="text">
            <string>osimage</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_fwdnx"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_fwimage"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_osdnx"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_osimage"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_fwdnx">
           <property name="text">
            <string>Chọn file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_fwimage">
           <property name="text">
            <string>Chọn file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_osdnx">
           <property name="text">
            <string>Chọn File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_osimage">
           <property name="text">
            <string>Chọn file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Button">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>528</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Start">
            <property name="text">
             <string>Bắt đầu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Stop">
            <property name="text">
             <string>Hủy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>pushButton_Start</zorder>
       <zorder>pushButton_Stop</zorder>
       <zorder>horizontalSpacer</zorder>
       <zorder>horizontalSpacer</zorder>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
